src/app/core/auth/auth-request.service.ts

```diff
@@ -139,4 +147,32 @@ export abstract class AuthRequestService {
       }),
     );
   }
+
+  /**
+   * Send a post request to create a machine token
+   */
+  public postToMachineTokenEndpoint(): Observable<RemoteData<MachineToken>> {
```

Comment:
I don't understand what these endpoints are. I didn't see any mention of `machinetokens` in the backend PR or REST Contract PR (https://github.com/DSpace/RestContract/pull/294)?  Is this an accidental change from some other feature?